variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+# List of variables related to power plants
+# Variables related to the CO2 emission by power plant
```

Comment:
please add all of these variables in `emissions.yaml` as subcategories of `Emissions|CO2|Energy|Supply|Electricity`

Reply:
ok